pkg/operators/wasm/testdata/perf/program.go

```diff
@@ -56,25 +55,25 @@ func gadgetStart() int32 {
 		return 1
 	}
 
+	for range 10 {
+		// Generate some events.
+		// While testing, this syscall returns an errno but the events are
+		// generated.
+		syscall.Open("/tmp/test-perf-array", syscall.O_CREAT|syscall.O_WRONLY|syscall.O_TRUNC, 0)
```

Comment:
I don't think it generates any event. The ebpf program is hooked on `tracepoint/syscalls/sys_enter_open` and `tracepoint/syscalls/sys_enter_openat` but this `syscall.Open` is running in the wasm module, so it can't run syscalls on the host kernel.

I see the tinygo `syscall.Open()` function is implemented with `libc_open()`:
https://github.com/tinygo-org/tinygo/blob/v0.25.0/src/syscall/syscall_libc.go#L68

I am not sure what it does, but still, it is running in the wasm sandbox, and I don't see where it would be connected to the host to perform syscalls on the host.